src/src_V2/modules/manager/financial/constants/payment-type.js

```diff
@@ -0,0 +1,8 @@
+const PaymentType = {
+  Cash: 1,
+  Transfer: 2,
+  1: 'Cash',
+  2: 'Transfer',
+};
+
+export default PaymentType;
```

Comment:
The constants sizes aren't so big i think we can put all this data in constants.js and remove files, how do you think?